ci-operator/step-registry/openshift/microshift/rebase/openshift-microshift-rebase-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+# print info on environment and tools
+echo "Environment:"
+printenv
+
+echo "Tool versions:"
+echo "  git: $(git version)"
+echo "  go: $(go version)"
+echo "  jq: $(jq --version)"
+echo "  oc: $(oc version)"
+
+oc get configmaps
+
+sleep 1800
+
+# fetch pull-secret for the central CI registry
+mkdir -p ~/.docker
+oc registry login --registry-config="${HOME}/.docker/config.json"
+
+TARGET_RELEASE_IMAGE="quay.io/openshift-release-dev/ocp-release:4.10.18-x86_64"
```

Comment:
Why not using `TARGET_RELEASE_AMD64`?

Reply:
That was an experiment. Please see the updated PR.